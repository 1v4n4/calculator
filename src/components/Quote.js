import React from 'react';
import '../CSS/quote.css';

const Quote = () => (
  <div className="quoteDiv">
    <h3>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding. â€“William Paul Thurston
    </h3>
  </div>
);

export default Quote;
