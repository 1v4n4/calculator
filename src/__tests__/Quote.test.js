import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders Quote page correctly', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
