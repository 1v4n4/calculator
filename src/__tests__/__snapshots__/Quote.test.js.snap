// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quote page correctly 1`] = `
<div
  className="quoteDiv"
>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h3>
</div>
`;
